<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>701</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="cardGridLayout">
     <property name="verticalSpacing">
      <number>4</number>
     </property>
    </layout>
   </widget>
   <widget class="QLabel" name="inTurnLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="statsTextBrowser">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>10</y>
      <width>381</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>375</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="startGridLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>19</number>
     </property>
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <item row="1" column="0">
      <widget class="QComboBox" name="cardComboBox"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="playerAmountLabel">
       <property name="text">
        <string>Number of players</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QLineEdit" name="playerNameLineEdit">
       <property name="maxLength">
        <number>52</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="playerComboBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="cardLabel">
       <property name="text">
        <string>Number of cards</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="playerNameLabel">
       <property name="text">
        <string>Player names (only A-Z/a-z, separated with space)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="closePushButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>520</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="continuePushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>520</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closePushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>711</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
